packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -132,11 +133,12 @@ class FluidAppInsightsLogger implements ITelemetryBaseLogger {
 	 */
 	private readonly baseLoggingClient: ApplicationInsights;
 	private readonly config: FluidAppInsightsLoggerConfig;
+
 	public constructor(client: ApplicationInsights, config?: FluidAppInsightsLoggerConfig) {
 		this.baseLoggingClient = client;
 		// Deep copy config to prevent issues if user mutates the object they passed in
 		this.config = config
-			? structuredClone(config)
+			? (structuredClone(config) as FluidAppInsightsLoggerConfig)
```

Comment:
We should avoid `as` as much as we can.
Remove the cast and add the `@types` package for `@ungap` - see `@fluidframework/tree`.